ur_description/launch/view_ur5.launch.py

```diff
@@ -3,6 +3,7 @@
 from launch import LaunchDescription
 from launch_ros.actions import Node
 from ament_index_python.packages import get_package_share_directory
+import xacro
```

Comment:
nitpick: please move this up with the other `import`s above and alphabetize them. So it should be:

import os
import xacro
import yaml
from launch import get_package_share_directory
from launch import LaunchDescription
from launch import Node